src/_data/glossary.yml

```diff
@@ -371,6 +385,32 @@
     - "language"
     - "libraries"
 
+- term: "Quick fix"
+  short_description: |-
+    An automated, local code edit targeted at fixing the issue reported by a specific diagnostic.
+  related_links:
```

Comment:
I don't know whether it would be useful for users, but we could consider adding a link to the [Diagnostic Messages](https://dart.dev/tools/diagnostic-messages) page.